server/settings.py

```diff
@@ -162,6 +162,7 @@ def env(variable, fallback_value=None):
     'superdesk.upload',
     'superdesk.notification',
     'superdesk.activity',
+    'superdesk.vocabularies',
```

Comment:
Nop, it's not like that. The `settings.py` file from sd-core is not shared at all, that is used only for setting up the app for testing purposes.
Each app has to define complete settings as we don't want to impose on apps what modules to use.